website/server/controllers/api-v3/groups.js

```diff
@@ -648,38 +648,64 @@ async function _inviteByEmail (invite, group, inviter, req, res) {
  *
  * @apiParam {String} groupId The group _id ('party' for the user party and 'habitrpg' for tavern are accepted)
  *
- * @apiParam {Array} [emails] Body parameter - An array of emails addresses to invite
+ * @apiParam {Array} [emails] Body parameter - An array of objects, each
+ * representing one email address to invite
+ * @apiParam {Object} [emails[0]] Key 'email' is required. Value must be a valid
+ * email address. Key 'name' is optional, value is invitee's name.
  * @apiParam {Array} [uuids] Body parameter - An array of uuids to invite
  *
  * @apiParamExample {json} Emails
- *   {
- *     "emails": ["user-1@example.com", "user-2@exmaple.com"]
- *   }
+ * {
+ *    "emails": [
+ *      {"email": "user-1@example.com"},
+ *      {"name": "User2", "email": "user-2@example.com"}
+ *    ]
+ *}
  * @apiParamExample {json} User Ids
  *   {
  *     "uuids": ["user-id-of-existing-user", "user-id-of-another-existing-user"]
  *   }
  * @apiParamExample {json} User Ids and Emails
- *   {
- *     "emails": ["user-1@example.com", "user-2@exmaple.com"],
- *     "uuids": ["user-id-of-existing-user", "user-id-of-another-existing-user"],
- *   }
+ * {
+ *   "emails": [
+ *       {"email": "user-1@example.com"},
+ *       {"email": "user-2@example.com"}
+ *   ],
+ *   "uuids": ["user-id-of-existing-user"]
+ *}
  *
- * @apiSuccess {Array} data The invites
+ * @apiSuccess {Array} data The invitees
  * @apiSuccess {Object} data[0] If the invitation was a user id, you'll receive back an object. You'll recieve one Object for each succesful user id invite.
  * @apiSuccess {String} data[1] If the invitation was an email, you'll receive back the email. You'll recieve one String for each successful email invite.
  *
- * @apiSuccessExample {json} Successful Response
+ * @apiSuccessExample {json} Successful Response with Emails
+ * {
+ *   "success": true,
```

Comment:
I see that I missed the Notification property. Just to clarify, do you mean the success and notification properties should be at the top of the file (representing all of the Groups API calls), or together with the other properties (ex. line 676)